app/tests/e2e.test.ts

```diff
@@ -0,0 +1,115 @@
+import fetch from "node-fetch";
+const baseUrl = "http://localhost:5001/helloworld";
+const testId = "test-id-for-raml";
+const transformedTestId = "test-with-transformer";
+
+const httpRequest = async <T extends { message: string }>({
+  method,
+  content,
+  request = baseUrl,
+}: {
+  method: "post" | "put" | "get" | "patch" | "delete";
+  content?: string;
+  request?: string;
+}): Promise<{ status: number; message?: T["message"] }> => {
+  const response = await fetch(request, {
+    method,
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: content,
+  });
+
+  const body =
+    response.status === 204
+      ? response.statusText
+      : (<T>await response.json()).message;
+
+  return { status: response.status, message: body };
+};
+
+describe("Api tests without transformer", () => {
+  it("should get helloworld", async () => {
+    const { status, message } = await httpRequest({ method: "get" });
+
+    expect(status).toBe(200);
+    expect(message).toBe("Hello world");
+  });
+
+  it("should post helloworld", async () => {
+    const content = {
+      message: "Hello world posted",
+    };
+    const { status, message } = await httpRequest({
+      method: "post",
+      content: JSON.stringify(content),
+    });
```

Comment:
Instead of redefining the const several times through the test, it may be better to move to a function
```
const content = (method: string) => JSON.stringify( {message: `Hello world ${method}`} );
```

```
    const method: HttpMethod = "put";
    const { status, message } = await httpRequest({
      method,,
      request: `${baseUrl}/${testId}`,
      content: content(method),
    });
```

This will also require you move the http method type def from the parameter in the httpRequest function to a individual type

Reply:
Moved